teams/adcs/LSM3030AGR_mag_driver.c

```diff
@@ -0,0 +1,53 @@
+/*
+ * LSM303AGR_mag_driver.c
+ *
+ *  Created on: Jan 30, 2020
+ *      Author: shubhkarmansingh
+ */
+
+#include "LSM303AGR_magnetometer.h"
+
+/* Imported function prototypes ----------------------------------------------*/
+extern uint8_t LSM303AGR_MAG_io_write(void *handle, uint8_t WriteAddr, uint8_t *pBuffer, uint16_t nBytesToWrite);
+extern uint8_t LSM303AGR_MAG_io_read(void *handle, uint8_t ReadAddr, uint8_t *pBuffer, uint16_t nBytesToRead);
```

Comment:
Were are you importing these functions from?  They also don't need to be `extern`